### PR TITLE
Fix value of lookup sync period in docs

### PR DESCRIPTION
**Pull request by kfaraz**
### Changes
- Update value of `druid.manager.lookups.period` in docs to `120_000`, which is the value used in `LookupCoordinatorManagerConfig`
- Remove statements that are not pertinent anymore

### CODE REVIEW

File: docs/configuration/index.md [hunk 1]
```diff
@@ -980,7 +980,7 @@ http://<COORDINATOR_IP>:<PORT>/druid/coordinator/v1/config/history?count=<n>
 ```
 
 ##### Lookups Dynamic Configuration
-These configuration options control the behavior of the Lookup dynamic configuration described in the [lookups page](../querying/lookups.md)
+These configuration options control the lookup management done by the coordinator. Configurations that affect propagation of lookups can be found in [dynamic configuration for lookups](../querying/lookups.md#dynamic-configuration).
```
Comment (techdocsmith) [inline docs/configuration/index.md:983]:
```suggestion
These configuration options control Coordinator lookup management. See [dynamic configuration for lookups](../querying/lookups.md#dynamic-configuration) configurations that affect lookup propagation.
```

File: docs/configuration/index.md [hunk 2]
```diff
@@ -989,7 +989,7 @@ These configuration options control the behavior of the Lookup dynamic configura
 |`druid.manager.lookups.deleteAllTimeout`|How long to wait for all `DELETE` requests to finish before considering the delete attempt a failure|PT10S|
 |`druid.manager.lookups.updateAllTimeout`|How long to wait for all `POST` requests to finish before considering the attempt a failure|PT60S|
 |`druid.manager.lookups.threadPoolSize`|How many processes can be managed concurrently (concurrent POST and DELETE requests). Requests this limit will wait in a queue until a slot becomes available.|10|
-|`druid.manager.lookups.period`|How many milliseconds between checks for configuration changes|30_000|
+|`druid.manager.lookups.period`|How many milliseconds between checks for configuration changes|120000 (2 mins)|
```
Comment (techdocsmith) [inline docs/configuration/index.md:992]:
```suggestion
|`druid.manager.lookups.period`|Number of milliseconds between checks for configuration changes|120000 (2 minutes)|
```

**Review by techdocsmith — CHANGES_REQUESTED** [review]
@kfaraz request some small stylistic changes

**Comment by kfaraz** [comment]
Thanks for the review, @techdocsmith !

**Review by techdocsmith — APPROVED** [review]
LGTM

**Pull request merged**